### PR TITLE
Announcement messaging

### PR DESCRIPTION
**Pull request by LauraDuRy**
## What does this PR do?

(Provide a description of what this PR does.)

## Test Plan

(Write your test plan here. If you changed any code, please provide us with clear instructions on how you verified your changes work.)

## Related PRs and Issues

(If this PR is related to any other PR or resolves any issue or related to any issue link all related PR and issues here.)

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

(Write your answer here.)

### CODE REVIEW

File: src/routes/blog/post/announcing-messaging-set-up-a-messaging-service [hunk 1]
```diff
@@ -0,0 +1,73 @@
+---
+layout: post
+title: "Announcing messaging: set up a full-functioning messaging service"
+description: 
+date: 2024-02-26
+cover: /images/blog/messaging-announcement.png
+timeToRead: 4
+author: eldad-fux
+category: engineering
```
Comment (eldadfux) [inline src/routes/blog/post/announcing-messaging-set-up-a-messaging-service:9]: It should be `product` or something. Engineering needs to be much more technical stuff about our internal processes. We need to have a clear list of categories for the future.

File: src/routes/blog/post/announcing-messaging-set-up-a-messaging-service [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+With Messaging, you can deliver messages to groups of users through topics or address individual users and their devices through targets. Messaging deeply integrates with integrated existing accounts in Appwrite out of the box. You can immediately start messaging your existing users and integrate deeply with other Appwrite features.
+
+You can address recipients of messages at three levels:
+
+- Topics, which hold a group of users’ devices.
+- Users which holds all targets associated with their account.
+- Targets, which are individual emails, phone numbers, and devices.
+
+You can publish announcements and newsletters through topics, deliver user-specific promotions and alerts to all users’ devices, and address individual targets for location-based or real-time alerts.
```
Comment (eldadfux) [inline src/routes/blog/post/announcing-messaging-set-up-a-messaging-service:49]: again devices is not a great word, it usually more common to use it for just push notifications.

File: src/routes/blog/post/announcing-messaging-set-up-a-messaging-service [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+Draft, schedule, and send your messages in the Appwrite Console, or use Appwrite SDKs to send messages through code.
+
+Drafting directly in the Appwrite Console is great for promo messages, newsletters, and other announcements. You can see a preview of what your message will look like right in the console and control how your message will look before sending.
+
+For real-time alerts, you can programmatically send messages to your users and integrate with Appwrite Functions triggered by platform events. This allows you to deliver automated messages like invoices, login alerts, order confirmations, and new message alerts.
+
+## Get started with Messaging
+
+To get started with messaging, you connect a provider, add topics and targets, and send your first message today for your project.
```
Comment (eldadfux) [inline src/routes/blog/post/announcing-messaging-set-up-a-messaging-service:63]: A paragraph about many more adapters to come and call to the community to contribute more could be helpful here.

File: src/routes/blog/post/announcing-messaging-set-up-a-messaging-service [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+- Users which holds all targets associated with their account.
+- Targets, which are individual emails, phone numbers, and devices.
+
+You can publish announcements and newsletters through topics, deliver user-specific promotions and alerts to all users’ devices, and address individual targets for location-based or real-time alerts.
+
+![Creating a message from the Console](/images/blog/message-overview.png)
+
+## Deliver messages your way
+
+Draft, schedule, and send your messages in the Appwrite Console, or use Appwrite SDKs to send messages through code.
```
Comment (eldadfux) [inline src/routes/blog/post/announcing-messaging-set-up-a-messaging-service:55]: I would love a bit more detail about the scheduling feature. Maybe mentioning some example use cases.

**Review by eldadfux — CHANGES_REQUESTED** [review]

File: src/routes/blog/post/announcing-messaging-set-up-a-messaging-service [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+
+For real-time alerts, you can programmatically send messages to your users and integrate with Appwrite Functions triggered by platform events. This allows you to deliver automated messages like invoices, login alerts, order confirmations, and new message alerts.
+
+You can send messages immediately or schedule them for future delivery. The Appwrite Console allows you to set dates and times from the UI easily. This is a great addition for use cases like appointment reminders, alerts, promotional messages, and any time you need to deliver a message at a specific time.
+
+## Get started with Messaging
+
+To get started with messaging, you connect a provider, add topics and targets, and send your first message today for your project.
+
+We will be adding more adopters over time and are open to contributions. Visit our [community page](https://appwrite.io/community) to learn more about contributing or add your issues directly to [GitHub](https://github.com/appwrite/appwrite/issues).
```
Comment (adityaoberai) [inline src/routes/blog/post/announcing-messaging-set-up-a-messaging-service:67]:
```suggestion
We will be adding more adapters over time and are open to contributions. Visit our [community page](https://appwrite.io/community) to learn more about contributing or add your issues directly to [GitHub](https://github.com/appwrite/appwrite/issues).
```

There was a type

**Review by gewenyu99 — APPROVED** [review]

**Pull request merged**